src/components/page.jsx

```diff
@@ -82,7 +82,13 @@ class Page extends React.Component {
         className={this.props.viewMode === 'presentation' ? 'presentation-mode' : ''}
       >
         <NotebookHeader />
-        <div id="cells" className={this.props.viewMode + (this.props.sidePane ? ' pane-open' : '')}>
+        <div
+          id="cells"
+          className={this.props.viewMode}
+          style={{
+            width: this.props.sidePane ? `calc(100% - ${this.props.sidePaneWidth + 17}px)` : '100%',
```

Comment:
this actually breaks quite a bit of css for presentation view. Can make sure this width only happens when we are in editor mode as well? Setting `div#cells` to `100%` for presentation view will counteract some internal reports, custom styling, etc. that others are relying on for their own presentation style sheets, which will set this `div` to be another width in some cases. (I'm noticing this because of an internal Mozilla notebook I test things against is broken with the change).

One approach would be to set the `width` to `undefined` in this operator if `this.props.sidePane === false` and `this.props.viewMode !== 'editor`.